pallets/api/Cargo.toml

```diff
@@ -11,8 +11,12 @@ targets = ["x86_64-unknown-linux-gnu"]
 
 [dependencies]
 codec.workspace = true
+log.workspace = true
 scale-info.workspace = true
 
+# Local
+pop-chain-extension.workspace = true
```

Comment:
This is a side effect of trying to isolate anything Pop API related, reusable across runtimes, within pallet-api.

The effectively makes the pop-api crate the ink! interface, with pallet-api the runtime implementation, with submodules added over time for each use case.